frontend/src/components/teacher/student-management/view-students/StudentsTable.tsx

```diff
@@ -8,15 +8,21 @@ import EditStudentPopover from "./EditStudentPopover";
 
 interface StudentsTableProps {
   students: StudentResponse[];
+  classId: string;
+  isClassActive: boolean;
 }
 
 const StudentsTable = ({
   students,
+  classId,
+  isClassActive,
 }: StudentsTableProps): React.ReactElement => {
   const headers = ["First Name", "Last Name", "Student ID"];
   const rows: TableRow[] = students.map((student) => ({
     values: [student.firstName, student.lastName, student.studentNumber],
-    menu: <EditStudentPopover />,
+    menu: isClassActive ? (
+      <EditStudentPopover classId={classId} student={student} />
```

Comment:
`EditStudentPopover` sounds like its only for edit, but idk what else you could call it. ManageStudentPopover sounds weird